activerecord/lib/active_record/fixtures.rb

```diff
@@ -761,13 +773,25 @@ def column_names
         @column_names ||= @connection.columns(@table_name).collect(&:name)
       end
 
-      def read_fixture_files(path, model_class)
+      def model_class=(class_name)
```

Comment:
I was avoiding setting `model_class` in `read_fixture_files` directly to the instance variable and having to `safe_constantize` there. It's easy to revert if preferable.